content/blog/_posts/inclusive-infra.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Towards More Inclusive Infrastructure"
+date: 2021-04-14
+slug: inclusive-infra
+authors: Reinhardt Quelle
```

Comment:
If you change this to 'author' it should display properly.

Reply:
done